docs/antora/modules/ROOT/pages/Getting_Started.adoc

```diff
@@ -2,26 +2,28 @@
 
 == Simple Project
 
-The simplest Mill build for a Java project looks as follows:
```

Comment:
Re-ordered this to make Java second after Scala, and removed the multi-module Java example since people can probably figure it out from the single-module java and multi-module scala examples